.scrutinizer.yml

```diff
@@ -0,0 +1,154 @@
+build:
+  tests:
+    override:
+      - flake8 --filename=* bin
+       - nosetests theano/tests/test_flake8.py
```

Comment:
I think it's actually the extra space that is throwing Scrutinizer off, not the comments.